test/application_system_test_case.rb

```diff
@@ -1,5 +0,0 @@
-require "test_helper"
-
-class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
-  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
-end
```

Comment:
Please remove this file.

Reply:
This is already removed such as whole test dir.

Reply:
sorry, right